projects/fsastorefrontlib/src/cms-components/checkout/components/quote-review/quote-review.component.html

```diff
@@ -271,6 +271,7 @@ <h2 class="row no-gutters section-header-heading">
             <div
               *ngIf="
                 cart.insuranceQuote?.quoteWorkflowStatus?.code === 'APPROVED' ||
+                cart.insuranceQuote?.quoteWorkflowStatus?.code === 'ACCEPTED' ||
```

Comment:
add final list that will contain accepted quote workflow status for this use case (continuing checkout)